### ISSUE TITLE
Design support - parking

### ISSUE THREAD

**Issue by sebastianebarb** [CityOfBoston/design-team-workflow#518]
Contact: Daniel Lesser <daniel.lesser@boston.gov>
Monum and Analytic are supporting the City’s performance based parking project and we are coming to the final stretch of the pilot. We’d like to produce a fairly detailed report on the findings for public consumption but could use some graphic design support to create some custom graphics based upon the data and help to layout the report for digital. Can we tap members of your team to support? Aiming for completion by early December.

1) We committed to a public document by the New Year
2)  At most 6 visualizations -- Michelle from the data team as well myself has started doing some visualizations 
3) We would need some support for digital lay out (we are not planning create a hard copy). The report is approximately 10 pages.

All files:
https://drive.google.com/open?id=1hSM9hIRSTXqYhAvJ3CKoQqz4FzBAgDmz

**Comment by sebastianebarb** [comment]
Text and support https://docs.google.com/a/boston.gov/document/d/1drBEW8DlZCR5qH20RNVkkTqvA6hQpkmh60QxeFFG3x0/edit?usp=sharing

**Comment by sebastianebarb** [comment]
I had a meeting with Chief Osgood and Commissioner Fiandaca on Thursday. They decided they wanted to split the report into two different reports. 

The first report would include everything until #5 Next Steps.  The second report would just include the #5 next steps section. I re-wrote the next steps section. Here it is. 

 Hopefully, this change won't create too many problems. Sebastian, I'll give you a call when you get back tomorrow. 

Best,
Dan

**Comment by carolinestj** [comment]
Switch x & y axis for graphs 

Change color order for Back Bay & seaport parking graph, so the highest concentration has red color and then down the color spectrum as the concentration reduces

new graph showing cars parked (waiting on content for that) 

check text for double sentences and make sure things are bolded where they need to be bolded

Add New titles for Back Bay & Seaport parking graphs  

new text should be done on the 8th of january 
final report tentatively due Jan 15th
